driver/src/price_finding/linear_optimization_price_finder.rs

```diff
@@ -168,3 +178,275 @@ fn read_output() -> std::io::Result<serde_json::Value> {
     let value = serde_json::from_reader(reader)?;
     Ok(value)
 }
+
+#[cfg(test)]
+pub mod tests {
+
+    use super::*;
+    use std::error::Error;
+
+    #[test]
+    fn test_solver_keeps_prices_from_previous_result() {
+        let state = models::State {
+            state_hash: "hash".to_string(),
```

Comment:
is this really supposed to be a string with the word hash in it? Or was this a place holder?

Reply:
It's a unit test, so yeah it's just a dummy string.